src/couch_file.erl

```diff
@@ -531,27 +537,96 @@ find_header(Fd, Block) ->
     {ok, Bin} ->
         {ok, Bin};
     _Error ->
-        find_header(Fd, Block -1)
+        find_header_pread2(Fd, Block -1)
```

Comment:
@davisp, I'm confused about this comment. There's already a `find_header/2` that's still being used to try to load a header at eof. If that fails, we fall back to the vectored read version, which is also arity 2, and so has to have some other name. How about `find_header_vectored/2`?